service/lib/agama/storage/config_conversions/to_model_conversions/with_partitions.rb

```diff
@@ -29,15 +29,11 @@ module ToModelConversions
         module WithPartitions
           # @return [Array<Hash>]
           def convert_partitions
-            valid_partitions
+            config.partitions
+              .reject(&:skipped?)
```

Comment:
Shouldn't this use `config.valid_partitions`?

Reply:
Not really. I thought it was a `Config` object. But actually it may make some sense.